src/test/java/org/folio/readingroom/utils/HelperUtils.java

```diff
@@ -30,4 +34,22 @@ public static ServicePoint createServicePoint(UUID servicePointId, String servic
     servicePoint.setName(servicePointName);
     return servicePoint;
   }
+
+  public static ReadingRoomServicePointEntity createServicePointEntity(UUID servicePointId, String servicePointName) {
+    ReadingRoomServicePointEntity servicePointEntity = new ReadingRoomServicePointEntity();
+    servicePointEntity.setId(servicePointId);
+    servicePointEntity.setName(servicePointName);
+    return servicePointEntity;
+  }
+
+  public static ReadingRoomEntity createReadingRoomEntity() {
+    ReadingRoomEntity readingRoomEntity = new ReadingRoomEntity();
+    readingRoomEntity.setId(READING_ROOM_ID);
+    readingRoomEntity.setName(READING_ROOM_NAME);
+    readingRoomEntity.setIspublic(true);
+    Set<ReadingRoomServicePointEntity> servicePoints = new HashSet<>();
+    servicePoints.add(createServicePointEntity(SERVICE_POINT_ID1, SERVICE_POINT_NAME1));
+    readingRoomEntity.setServicePoints(servicePoints);
+    return readingRoomEntity;
+  }
```

Comment:
Can we have test cases where we are creating multiple reading rooms in one request ?